.travis.yml

```diff
@@ -1,25 +1,33 @@
 language: scala
+cache:
+  directories:
+    - $HOME/.ivy2
 scala:
    - 2.10.4
-before_install:
-  - sudo pip install --upgrade conda py4j
```

Comment:
Note that I explicitly do not install Py4J anymore, since forcing an upgrade seemed to break things.